terraform/modules/jenkins/log_bucket/variables.tf

```diff
@@ -0,0 +1 @@
+variable "name" {}
```

Comment:
So is this `name` the thing we supply when we define `module "jenkins_elb_log_bucket"` in `main.tf`?

Reply:
Yup